tensorflow_datasets/core/features/features_dict.py

```diff
@@ -138,7 +139,7 @@ def __iter__(self):
 
   def __repr__(self):
     """Display the feature dictionary."""
-    return '{}({})'.format(type(self).__name__, self._feature_dict)
+    return pprint_features_dict(self._feature_dict, types='FeaturesDict')
```

Comment:
Instead of calling `pprint_features_dict` and have the `kwargs hack which seems hacky`, I feel it would be nicer to have FeaturesDict simply calling `repr()` recursively on its child.

Something like:
```python
def __repr__(self):
  lines = ['{}({'.format(type(self).__name__)]
  for key, feature in sorted(list(features_dict.items())):
    all_sub_lines = '\'{}\': {},'.format(key, feature)
    lines.extend('    ' + l for l in all_sub_lines.split('\n'))  # Add indentation to all childs
  lines.append('})')
  return lines.join('\n')
```

This is more generic as this would works with any feature which is using multiple line. And it avoid having to test `insinstance(, FeatureDict)` or `hasattr`.
What do you think ?

Reply:
I'm agree with you but now we don't use the `pprint_features_dict` func.